javascript/Task3/task3.js

```diff
@@ -0,0 +1,33 @@
+let btn = document.getElementById('btn-seconds')
+btn.addEventListener("click", printMessage);
+let messagePrint = null;
+let print = null;
+let clicked = false;
+const message = "Hello";
+let second = null;
+
+function printMessage () {
+    if(!clicked) {
+        second = Math.round(Math.random() * (4 - 1) + 1);
+        printSeconds(second);
+        messagePrint = setTimeout(function() {
+            console.log(message);          
+        }, second * 1000);
+        clicked = true;
+    }
+    else {
+        clearTimeout(messagePrint);
+        for(let i=print.length; i>0; i--) {
+            clearTimeout(print);
+        }
+        clicked = false;
+    }
+}
+
+function printSeconds(second) {
+    for(let i=second, temp=0; i > 0; i--, temp++) {
```

Comment:
how about starting the single timer which will print seconds every second instead of starting an array of them?